### ISSUE TITLE
Clarify the usage of the default forms template

### ISSUE THREAD

**Issue by morhi** [statamic/docs#138]
When looking at the **default forms template** at https://statamic.dev/forms#the-template I recognized, that there is a new line compared to the Statamic v2 docs:

```html
<div class="p-1">{{ field }}</div>
```

So I wondered what this line does, because I did not find any hint on that page. After trying out the code it turned out that the `field` property renders the complete input field. That is nice, so far. However, I wanted to give the input an extra class. In order to find out how that would work I look at the code behind.

My first thought was that `field` must be a special **Tag** which renders the field. But that is not true. The `field` attribute is actually an instance of `\Illuminate\View\View`, which returns the `html` markup of a partial. When looking at the partials at `vendor/statamic/cms/resources/views/extend/forms/` I realized, that it is not easily possible to change the content of the partial, but that it is better to create my own frontend partials again, as I already did in Statamic v2. Also, when trying to use something different than `Text` or `Textarea` like `Checkbox`, it would still generate a plain text input field.

Long story short, I'd suggest to change this line in the docs to something like this, to prevent people from asking themselves how to modify the generated field markup:

```html
{{ if type === 'text' }}
    <input type="{{ input_type ?? 'text' }}" name="{{ handle }}" value="{{ old }}" />
{{ /if }}

{{ if type === 'textarea' }}
    <textarea name="{{ handle }}" rows="5">{{ old }}</textarea>
{{ /if }}

{{ if type === 'checkboxes' }}
    {{ foreach:options as="key|label" }}
        <div>
            <input type="checkbox" name="{{ handle }}[{{ key }}]" value="1" />
            <span>{{ label }}</span>
        </div>
    {{ /foreach:options }}
{{ /if }}
```

What do you think?

**Comment by jackmcdade** [comment]
Good catch on this one – there should either be a note or a simplified example. Let me put my UX hat on and decide :)

For what it's worth, here's where that bit is explained: https://statamic.dev/tags/form-create#dynamic-rendering

**Comment by jackmcdade** [comment]
Improved in c6369aa. Thanks for the nudge!
